plugins/plugin-chart-choropleth-map/src/chart/ChoroplethMap.tsx

```diff
@@ -58,31 +60,30 @@ export type ChoroplethMapVisualProps = {
   map?: string;
 };
 
-export type ChoroplethMapProps = ChoroplethMapVisualProps & {
-  data: Record<string, unknown>[];
-  height: number;
-  width: number;
-};
+export type ChoroplethMapProps = ChoroplethMapVisualProps &
+  WithTooltipProvidedProps<MapDataPoint> & {
+    data: Record<string, unknown>[];
+    height: number;
+    width: number;
+  };
 
 const defaultProps = {
   data: [],
   encoding: {},
   map: 'world',
 };
 
-const missingItem: ChoroplethMapChannelOutputs = DefaultChannelOutputs;
+const missingItem = DefaultChannelOutputs;
```

Comment:
Is this variable necessary?

Reply:
was planning to have a different style for missing data. Now using the default output as placeholder.